src/components/CovMap.tsx

```diff
@@ -294,12 +294,13 @@ export const CovMap = () => {
       <Legend />
       <Dialog
         aria-labelledby="simple-dialog-title"
-        open={!datasetFound}
+        open={!datasetFound || true}
         style={{
           zIndex: 1190,
           touchAction: "none",
           position: "absolute",
         }}
+        disableEnforceFocus
```

Comment:
Does this also mean that the keyboard won't open automatically on mobile?

Reply:
na it means if the second modal opens on top of the welcomeStepsModal it won't hijack the the focus from the welcomeStepsModal and it's input field.